src/gui/optionsdialog.cpp

```diff
@@ -704,6 +704,11 @@ void OptionsDialog::loadDownloadsTabOptions()
     connect(m_ui->groupMailNotifAuth, &QGroupBox::toggled, this, &ThisType::enableApplyButton);
     connect(m_ui->mailNotifUsername, &QLineEdit::textChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->mailNotifPassword, &QLineEdit::textChanged, this, &ThisType::enableApplyButton);
+    connect(m_ui->sendTestEmail, &QPushButton::clicked, this, [this]
+    {
+        app()->sendTestEmail();
+        QMessageBox::information(this, tr("Test email"), tr("Attempted to send email. Check your inbox to confirm success"));
```

Comment:
Shouldn't there be any punctuation mark at the end of the second sentence?

Reply:
From looking at other messages, no. They generally omit punctuation entirely, and in the case there are multiple sentences, the last omits punctuation.

Reply:
> From looking at other messages, no. They generally omit punctuation entirely, and in the case there are multiple sentences, the last omits punctuation.

Is it a good idea to follow the existing incorrect things?
Well, I'm not insisting. They all can be fixed later.